spec/resources/sites_spec.rb

```diff
@@ -3,13 +3,42 @@
 describe TableauApi::Resources::Sites, vcr: { cassette_name: 'sites' } do
   include_context 'tableau client'
 
+  let(:default_client) do
+    TableauApi.new(
+      host: ENV['TABLEAU_HOST'],
+      site_name: 'Default',
+      username: ENV['TABLEAU_ADMIN_USERNAME'],
+      password: ENV['TABLEAU_ADMIN_PASSWORD']
+    )
+  end
+
+  describe '#create' do
+    it 'can create a site' do
+      site = default_client.sites.create(name: 'Test Site 2', content_url: 'TestSite2', admin_mode: 'ContentAndUsers')
+      expect(site).not_to be_nil
+      expect(site['id']).to be_a_tableau_id
+      expect(site).to eq(
+        'id' => site['id'],
+        'name' => 'Test Site 2',
+        'contentUrl' => 'TestSite2',
+        'adminMode' => 'ContentAndUsers',
+        'state' => 'Active',
+        'disableSubscriptions' => 'true'
+      )
+    end
+
+    it 'raises an error if fails to create a site' do
+      expect do
+        default_client.sites.create(name: 'Test Site 3', content_url: 'TestSite2', admin_mode: 'ContentAndUsers')
```

Comment:
using a different name but same content url so that it fails but we don't hit an existing VCR episode in the cassette